### ISSUE TITLE
Testing, QA, & Fixes

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-ipbs#35]

**Comment by MirandaEcho** [comment]
### Homepage Desktop 

- [ ] Double check fonts - headers look wrong
- [x] Reduce white space and tighten up site header, especially above and below logo as much as possible
- [x] Reduce the height of the featured image (crop placeholder if needed, client had concern about scroll depth for the top area)
- [x] Can the member station descriptive text be centered and constrained like the mockup?
- [x] Reduce size of member map on desktop 
- [x] Member station map key is missing (it wasn't in the desktop mockup, apparently, but was in mobile and is on the live site)
- [x] Reduce white space above and below member map embed
- [x] Decrease height (and overall size) of member list sections
- [x] Features section is broken - do we need to add content here to QA?
- [x] Add some space above and below video embed section
- [x] More events button missing
- [x] Can sponsors slider be wider?
- [x] Footer: Text color is white instead of very light gray
- [x] Footer: Can we reduce the font size for the member station links by a few px?

**Comment by MirandaEcho** [comment]
### Homepage Mobile

- [x] Mobile Header: Reduce white space above and below logo
- [ ] Featured image is a bit too small (height) - see mockup
- [x] Reduce white space above member stations

**Comment by benlk** [comment]
On fonts: Should we use the thin fonts from the early PDFs, which were screenshotted for many of the issues here, or the thick fonts from the Adobe XD designs, which were what got put on the site?

**Comment by MirandaEcho** [comment]
### Interior Page Template (desktop):

Note: Test Page to match mockup features: http://ipbs.staging.wpengine.com/?page_id=530

- [ ] remove byline & date from template
- [ ] Left align blocks overlay if placed too close together:
![Screen Shot 2020-02-04 at 7 53 44 PM](https://user-images.githubusercontent.com/48263361/73804024-16b05280-4788-11ea-92d0-dc87e3d3d1f5.png)
- [ ] Needs to be more space between blocks on left and text on right
- [ ] Missing spacer line in article prefooter between body text and sponsors section
    <img width="895" alt="Screen Shot 2020-02-04 at 7 56 27 PM" src="https://user-images.githubusercontent.com/48263361/73804137-6f7feb00-4788-11ea-89f0-7c9612d0a4eb.png">

- [x] Update URL slug rules to use page title to create slug, rather than just Page ID

**Comment by MirandaEcho** [comment]
### Interior page: Mobile
- [ ] Mobile menu displays over page title

**Comment by joshdarby** [comment]
> Features section is broken - do we need to add content here to QA?

Yes, this is a Largo Recent Posts widget. There are 0 published posts on the site, which is why nothing appears in that widget.

**Comment by joshdarby** [comment]
>  Double check fonts - headers look wrong

The design uses `Inter, Extra Bold`.

`h1-6` and `.widgettitle` use `'Inter',...` with `font-weight: 800;` as specified by the [Inter font repo guide](https://github.com/rsms/inter)

![Screen Shot 2020-02-05 at 9 39 46 AM](https://user-images.githubusercontent.com/18353636/73851443-7ac02e80-47fb-11ea-85ed-4d20e6a7fc9f.png)

**Comment by joshdarby** [comment]
> Footer: Text color is white instead of very light gray

The designs show the footer text color as `#FFF`

![Screen Shot 2020-02-05 at 10 00 45 AM](https://user-images.githubusercontent.com/18353636/73853162-6c274680-47fe-11ea-8ef1-995443df2604.png)

**Comment by joshdarby** [comment]
> Video is cropped horizontally on mobile

@MirandaEcho Do you have an example of this?

**Comment by benlk** [comment]
I'm seeing letterboxing, but that's kind of expected?  Unless there's an aspect ratio class we can use to make it keep the right shape.

![Screen Shot 2020-02-05 at 12 35 34](https://user-images.githubusercontent.com/1754187/73867095-0c3b9a80-4814-11ea-9cb0-6bb8905c8802.png)
![Screen Shot 2020-02-05 at 12 35 23](https://user-images.githubusercontent.com/1754187/73867093-0a71d700-4814-11ea-9f6f-0eed2361665b.png)

**Comment by joshdarby** [comment]
> I'm seeing letterboxing, but that's kind of expected?

That's what I'm seeing too.

**Comment by joshdarby** [comment]
> Featured image is a bit too small (height) - see mockup

In the design its height is 170px. On the actual site it's 315px.

**Comment by benlk** [comment]
> Update URL slug rules to use page title to create slug, rather than just Page ID 

I've updated the permalink structure on staging to `/%year%/%monthnum%/%day%/%postname%/`, which has the effect of making the page permalink structure be `/%postname%/`

**Comment by benlk** [comment]
The interior page revisions in https://github.com/INN/umbrella-ipbs/pull/84 came in at just over an hour, before review.
